src/sql/workbench/parts/dashboard/browser/dashboard.component.ts

```diff
@@ -4,6 +4,7 @@
  *--------------------------------------------------------------------------------------------*/
 
 import 'vs/css!./dashboard';
+import 'vs/css!./media/flexbox';
```

Comment:
do each of these files need the entire css contents? I'm pretty sure people have been using these styles as a crutch for awhile since they originally came from the extension.

Reply:
From what I saw yes, but I'm not an expert in this area.